macros/get_lookback_date_limits.sql

```diff
@@ -0,0 +1,68 @@
+{% macro get_lookback_date_limits(limit_type) %}
+  {{ return(adapter.dispatch('get_lookback_date_limits', 'snowplow_fractribution')(limit_type)) }}
+{% endmacro %}
+
+{% macro default__get_lookback_date_limits(limit_type) %}
+
+  -- check if web data is up-to-date
+
+  {% set query %}
+    with base as (
+      select max(start_tstamp) max_start_tstamp
+      from {{ var('page_views_source') }}
+    )
+
+    select
+      case when max_start_tstamp  < '{{ var('conversion_window_end_date') }}'
+      then True else False end as is_over_limit
+    from base
+  {% endset %}
```

Comment:
Do we need to consider the other possibility, that page_views could be run on the past month of data, for example, but then the user tries to run frac on the past 2 months, etc? this check would pass in that situation, from what i understand? I imagine that situation is much less likely though than what we're already checking for here.

Reply:
Good point, I forgot that in the end I decided not to use the manifest where we only have the upper boundary defined. will add the lower limit check as well.